msal/versioning/version.properties

```diff
@@ -1,3 +1,3 @@
 #Wed Aug 01 15:24:11 PDT 2018
-versionName=2.0.10
+versionName=2.0.12
```

Comment:
Why not 2.0.11?

Reply:
Because I fat-fingered the 2.0.11 branch name and am unable to fix it without basically disabling the force-push protection on GitHub.

Reply:
Not sure I understand correctly...seems like there is already a `release/2.0.11` on GitHub...can't we just change the base branch here? 

https://github.com/AzureAD/microsoft-authentication-library-for-android/compare/release/2.0.11...adjoh/release-2.0.11